tutorials/clinical/10_ieeg_localize.py

```diff
@@ -0,0 +1,505 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-ieeg-localize:
+
+========================================
+Locating Intracranial Electrode Contacts
+========================================
+
+Intracranial electrophysiology recording contacts are generally localized
+based on a post-implantation computed tomography (CT) image and a
+pre-implantation magnetic resonance (MR) image. The CT image has greater
+intensity than the background at each of the electrode contacts and
+for the skull. Using the skull, the CT can be aligned to MR-space.
+Contact locations in MR-space are the goal because this is the image from which
+brain structures can be determined using the
+:ref:`tut-freesurfer-reconstruction`. Contact locations in MR-space can also
+be translated to a template space such as ``fsaverage`` for group comparisons.
+"""
+
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#          Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+import nibabel as nib
+from dipy.align import (affine_registration, center_of_mass, translation,
+                        rigid, affine, resample)
+from dipy.align.reslice import reslice
+
+import mne
+from mne.datasets import fetch_fsaverage
+
+print(__doc__)
+
+np.set_printoptions(suppress=True)  # suppress scientific notation
+
+# paths to mne datasets - sample sEEG and FreeSurfer's fsaverage subject
+# which is in MNI space
+misc_path = mne.datasets.misc.data_path()
+sample_path = mne.datasets.sample.data_path()
+subjects_dir = op.join(sample_path, 'subjects')
+
+# use mne-python's fsaverage data
+fetch_fsaverage(subjects_dir=subjects_dir, verbose=True)  # downloads if needed
+
+###############################################################################
+# Aligning the T1 to ACPC
+# =======================
+#
+# Let's load our T1-weighted image and visualize it. As you can see, the
+# image is already aligned to the anterior commissue and posterior commissure
+# (ACPC). This is recommended to do before starting. This can be done using
```

Comment:
Why is this recommended? it doesn't seem like it should be necessary for the affine to be computed correctly. The center-of-mass alignment should render this step unnecessary

Reply:
I guess this is extraneous, it was in `img_pipe` so I thought there was a good reason for it but I guess `recon-all` works just as well without it.

Reply:
I would ask whoever wrote the recommendation what it's based on. If it's just "nice to have" then maybe we don't need it.

Or maybe better we can update the dataset (again!) to have the actual original, not-ACPC-aligned T1, plus a voxel number (found in FreeView) of the ACPC, and write a little function `img_realigned = mne.morph.realign_acpc(img, acpc_voxel)` that does this. That could be a follow-up PR, though :)

Reply:
(My naive sense is that this is probably just "good FreeSurfer practice", in which case it should live in some separate FreeSurfer tutorial and we should talk about this there. But if it's for example standard in iEEG to do this then it should live here.)